tailcfg/tailcfg.go

```diff
@@ -90,7 +90,8 @@ type CapabilityVersion int
 //   - 51: 2022-11-30: Client understands CapabilityTailnetLockAlpha
 //   - 52: 2023-01-05: client can handle c2n POST /logtail/flush
 //   - 53: 2023-01-18: client respects explicit Node.Expired + auto-sets based on Node.KeyExpiry
-const CurrentCapabilityVersion CapabilityVersion = 53
+//   - 54: 2023-01-19: Node.Cap added, PeersChangedPatch.Cap, uses Node.Cap for ExitDNS before Hostinfo.Services fallback
+const CurrentCapabilityVersion CapabilityVersion = 54
```

Comment:
May want to also change the comment for `CapabilityVersion` to indicate that it's used for more than just client <-> control negotiation. I had it as:

```
-// server (control plane), bump this number and document what's new.
+// server (control plane), peers (over PeerAPI) or frontend (over LocalAPI),
+// bump this number and document what's new.
```

Reply:
Good idea, done.